OracleUnifiedDirectory/dockerfiles/12.2.1.4.0/.gitignore

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Note that `All rights reserved.` is no longer used by Oracle. I'm not going to repeat this review comment for every instance, though.